packages/gestalt/src/icons/flash.svg

```diff
@@ -0,0 +1 @@
+<svg height="24" viewBox="0 0 24 24" width="24"><path d="M17.45 0L3.5 14h6.586L6.55 24 20.5 10h-6.586z" fill="#8e8e8e" fill-rule="evenodd"></path></svg>
```

Comment:
This SVG markup is invalid (see the file). You need to self close the `path` tag, did this file come from PDS? They should have given you the correct file so that you dont need to worry about this stuff
![image](https://user-images.githubusercontent.com/7877010/61250610-f7c77180-a70c-11e9-84af-25f0166f5ab9.png)